src/rotest/api/common/responses.py

```diff
@@ -0,0 +1,52 @@
+"""Responses models of the view requests."""
+from swaggapi.api.builder.common.response import AbstractResponse
+from swaggapi.api.builder.common.fields import (StringField,
+                                                ModelField,
+                                                ArrayField, BoolField)
+
+from rotest.api.common import GenericModel
+
+
+class SuccessResponse(AbstractResponse):
+    """Returns when request ended successfully - with no content."""
+    PROPERTIES = []
+
+
+class TokenResponseModel(AbstractResponse):
+    """Returns a token of the created sessions."""
+    TITLE = "Token Response"
+    PROPERTIES = [
+        StringField(name="token", description="Token of the current session",
+                    required=True)
+    ]
+
+
+class CleanupUserResponseModel(AbstractResponse):
```

Comment:
I don't understand this response
In addition, the documentation doesn't seem related

Reply:
?

Reply:
delete